pkg/cli/admin/release/info.go

```diff
@@ -734,16 +740,33 @@ func (i *ReleaseInfo) Platform() string {
 	return fmt.Sprintf("%s/%s", os, arch)
 }
 
-func (o *InfoOptions) LoadReleaseInfo(image string, retrieveImages bool) (*ReleaseInfo, error) {
+// LoadReleaseInfo takes an image and returns release payload info or an error. The release image will be looked up using
+// alternative image sources from image content source policy objects and/or from the user-given string, if setRef is true
+// If retrieveImages is true, release payload component images will also be fetched.
+func (o *InfoOptions) LoadReleaseInfo(image string, retrieveImages bool, setRef bool) (*ReleaseInfo, error) {
+	replacedRef := false
+	opts := extract.NewExtractOptions(genericclioptions.IOStreams{Out: o.Out, ErrOut: o.ErrOut})
```

Comment:
`opts := extract.NewExtractOptions(o.IOStreams)` will be sufficient

Reply:
updated, thanks ;)